Storage/rest/plain.h

```diff
@@ -293,6 +293,14 @@ struct Plain {
     }
   };
 
+  template <typename STORAGE>
+  struct RESTfulCQRSHandler {
+    template <typename F>
+    void Enter(Request request, F&& next) {
```

Comment:
Yes, a convention for Current Storage. All RESTful handlers have `Enter`. @nyospe's, closed-source, secured implementation of REST overrides these `Enter`-s to check auth, and I concluded the least error-prone way would be to make CQRS endpoints securable in the same fashion,